content/blog/projects/finished/adr.md

```diff
@@ -0,0 +1,41 @@
+---
+lastmod: 2023-06-17T10:13:20Z
+head:
+  meta:
+    - name: keywords
+      content: projects, robots, finished
+    - name: robots
+      content: index follow
+    - name: author
+      content: Andrzej Piotrowski
+    - name: copyright
+      content: © 2023 KN Integra
+    - name: createdAt
+      content: 2023-03-20T19:40:00Z
+---
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/e6f8639b-19d4-450a-836b-1aa75fe1f474" alt="Image description" style="max-height: 250px;  display: block; margin: 0; padding-left:160px">

```

Comment:
Use correct Markdown syntax.
If exact sizes are required, request a new component in the `integra.front` repo